.github/workflows/appengine-typescript.yaml

```diff
@@ -21,17 +21,17 @@ jobs:
       pull-requests: 'write'
       id-token: 'write'
     steps:
-    - uses: 'google-github-actions/auth@v0.3.1'
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{github.event.pull_request.head.ref}}
```

Comment:
Can you fetch the `github.event.pull_request.head.sha` instead of `ref`?

This ensures it fetches the source code at the sha when someone added the label, I think.

Reply:
Since this is a template, I created an issue to do this for all workflows at another time: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues/2589